schemas/public/spec-distribution-modules-auth-pomerium.json

```diff
@@ -0,0 +1,96 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "secrets": {
+      "$ref": "#/$defs/Spec.Distribution.Modules.Auth.Pomerium.Secrets"
+    },
+    "defaultRoutesPolicy": {
+      "type": "array",
+      "items": {
+        "type": "object"
+      }
+    },
+    "routes": {
+      "type": "array",
+      "items": {
+        "type": "object"
+      }
+    },
+    "policy": {
+      "type": "string"
+    },
+    "overrides": {
+      "$ref": "#/$defs/Types.FuryModuleComponentOverrides"
+    }
+  },
+  "required": ["secrets"],
+  "oneOf": [
+    {
+      "required": ["policy"]
+    },
+    {
+      "required": ["routes"]
+    }
```

Comment:
Why should this be required?

Reply:
To accept only policy or routes and not both, they are not required